backend/telemetry_shard/src/main.rs

```diff
@@ -359,19 +360,18 @@ where
                     let _ = tx_to_aggregator.send(FromWebsocket::Add {
                         message_id,
                         ip: real_addr,
-                        node: info.node,
+                        node: Box::new(info.node),
                         genesis_hash: info.genesis_hash,
                     }).await;
                 }
                 // Anything that's not an "Add" is an Update. The aggregator will ignore
                 // updates against a message_id that hasn't first been Added, above.
-                else {
-                    if let Some(last_seen) = allowed_message_ids.get_mut(&message_id) {
-                        *last_seen = Instant::now();
-                        if let Err(e) = tx_to_aggregator.send(FromWebsocket::Update { message_id, payload } ).await {
-                            log::error!("Failed to send node message to aggregator: {}", e);
-                            continue;
-                        }
+                else if let Some(last_seen) = allowed_message_ids.get_mut(&message_id) {
```

Comment:
I guess this requires a relative new rust compiler this is "new" syntax :)

Nice

Reply:
I believe `else if` was available from rust 1.0 :thinking: